samples/dgs-webmvc/src/main/java/sample/dgs/webmvc/DGSWebMvcSampleApplication.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017 Alexey Zhokhov
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package sample.dgs.webmvc;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+/**
+ * @author Adam Setch
+ */
+@SpringBootApplication(scanBasePackages = {"sample","com.tailrocks.graphql.datetime" })
```

Comment:
@donbeave - not really happy with having to set the base packages to scan this way so that the [DgsRuntimeWiring functions](https://netflix.github.io/dgs/scalars/?query=dgsruntimewiring#register-scalar-extensions-via-dgsruntimewiring).  Open to suggestions

Reply:
I found a way how to avoid it, use same approach as how official DGS register extended scalars: https://github.com/Netflix/dgs-framework/blob/b6f8894e1c9b338ad22fd68be5f3bd55eafc9558/graphql-dgs-extended-scalars/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsExtendedScalarsAutoConfiguration.kt

I have updated your branch, please check these changes.